documentation/manifest.md

```diff
@@ -82,6 +82,15 @@
         - [ ] Progress bar appears above Synchronization title
         - [ ] Team Explorer view changes to Home page
         - [ ] Notification appears: "Repository published successfully"
+        - [ ] Publish a private repo and verify on .com that it's private 
+- [ ] **Project section (Home button)*** 
+    - [ ] Click on "Home" icon 
+        - [ ] Verify Project has the following sections/buttons: Changes, Pull Requests, Branches, Sync, Pulse, Graphs, Wikis, Issues, Wikis, and Settings
```

Comment:
Thanks for adding this!

Wikis and issues will depend on whether the repository has them enabled, so they might not show up all the time.
The Changes, Branches, Sync and Settings buttons are from Microsoft, so probably no need to validate those. I imagine they aren't going anywhere, but that depends on what MS does with VS.